x/ecocredit/msgs_test.go

```diff
@@ -0,0 +1,741 @@
+package ecocredit
```

Comment:
could you add tests for `MsgSetPrecision`?

Reply:
I think we're going to remove `MsgSetPrecision` as part of #428, so might not be a good use of time to add them. What do you think @blushi?

Reply:
makes sense!